### PR TITLE
Edit db

### PR DESCRIPTION
**Pull request by MahmoudMH**
#37 #45

### CODE REVIEW

File: public/assets/github.svg [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="438.549px" height="438.549px" viewBox="0 0 438.549 438.549" style="enable-background:new 0 0 438.549 438.549;"
+	 xml:space="preserve">
+<g>
+	<path d="M409.132,114.573c-19.608-33.596-46.205-60.194-79.798-79.8C295.736,15.166,259.057,5.365,219.271,5.365
+		c-39.781,0-76.472,9.804-110.063,29.408c-33.596,19.605-60.192,46.204-79.8,79.8C9.803,148.168,0,184.854,0,224.63
+		c0,47.78,13.94,90.745,41.827,128.906c27.884,38.164,63.906,64.572,108.063,79.227c5.14,0.954,8.945,0.283,11.419-1.996
```
Comment (yahyaHB) [inline public/assets/github.svg]: What the fuck is this file?

File: src/client/components/admindashborad/students/index.js [hunk 2]
```diff
@@ -1,15 +1,18 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
 
 import './style.css';
 import PropTypes from 'prop-types';
 
 class Students extends Component {
   constructor(props) {
     super(props);
+    console.log(props,'ppp');
```
Comment (yahyaHB) [inline src/client/components/admindashborad/students/index.js]: remove the khara!

File: src/client/components/admindashborad/students/index.js [hunk 3]
```diff
@@ -48,23 +51,26 @@ class Students extends Component {
...
+                this.setState({ backgroundColor: 'orange' }):
+                this.setState({ backgroundColor: 'red' })
           }
           <div className='student__circle' title={this.props.dataStudent.status}
             style={this.state}></div>
-          <input className='student__view' type='submit' value='View'></input>
-        </div>
+          <Link to={`/profile/${this.props.dataStudent.username}`}>
+            <input className='student__view' type='submit' value='View'/>
+          </Link>  </div>
```
Comment (yahyaHB) [inline src/client/components/admindashborad/students/index.js]: nice one :+1:

File: src/client/components/navbar/Navbar.js [hunk 4]
```diff
@@ -3,7 +3,16 @@ import React, { Component } from 'react';
...
 class Navbar extends Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+  handleSubmit() {
+    this.props.propsNav.history.push('/profilesettings');
+  }
   render() {
+    console.log(this.props,'Naaav props');
```
Comment (yahyaHB) [inline src/client/components/navbar/Navbar.js]: remove all the friggin logs from all the friggin files

File: src/client/components/questions/q6.js [hunk 5]
```diff
@@ -43,6 +43,10 @@ class Q6 extends Component {
               <input className='q6__lastInput'type='text' placeholder='Link your portfolio' onChange={this.handlePortfolio}/>
               <h3 className='q6_input__h3'>Upload CV. <i className='fa fa-upload'></i></h3>
             </div>
+            <div className='q6_input'>
+              <input type='text' placeholder='Project1 Title' onChange={this.handleLinks}/>
+              <input type='text' placeholder='Project2 Title' onChange={this.handleLinks}/>
+            </div>
```
Comment (yahyaHB) [inline src/client/components/questions/q6.js]: I think making map is better

File: src/client/components/studentProfile/index.js [hunk 6]
```diff
@@ -12,26 +12,32 @@ import { BeatLoader } from 'react-spinners';
...
 
     return (
       <div>
-        <Navbar />
+
         <div>
           {
             isFetching
-              ? <center className='center'>
+              ? <center className='center' style={{ marginTop: '250px' }}>
```
Comment (yahyaHB) [inline src/client/components/studentProfile/index.js]: style={{ marginTop: '250px' }}
what?!!!
do it right

File: src/server/controllers/settingsData.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+  view.studentData(name,(dataBaseConnectionErorr, studentData) => {
+    if (dataBaseConnectionErorr) res.status(500).send(dataBaseConnectionErorr);
+    else {
+      if (studentData.length === 0) {
+        res.send('No Student Data');
+      } else {
+        res.send(studentData);
+      }
+    }
+  });
```
Comment (yahyaHB) [inline src/server/controllers/settingsData.js]: I can't understand the job of this file

File: src/server/controllers/updateDataStudent.js [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+exports.post = (req, res) => {
+  console.log(req.session.passport.user);
+  console.log(req.body);
+  //
+  // const student_id = req.session.passport.user.id;
+  //
+  // const dataToUpdate = req.body.data;
+  //
+  // // Assume data reached as
+  // student_id = 33;
```
Comment (yahyaHB) [inline src/server/controllers/updateDataStudent.js]: and what exactly is this file?

Comment (MahmoudMH) [reply src/server/controllers/updateDataStudent.js]: we have not finished update data students yet

File: src/server/controllers/settingsData.js (hunk 7 above)
Comment (MahmoudMH) [reply src/server/controllers/settingsData.js]: we have not finished update data students yet

File: src/client/components/studentProfile/index.js (hunk 6 above)
Comment (MahmoudMH) [reply src/client/components/studentProfile/index.js]: Ok i did

File: src/client/components/questions/q6.js (hunk 5 above)
Comment (MahmoudMH) [reply src/client/components/questions/q6.js]: it does not work on map

File: src/client/components/navbar/Navbar.js (hunk 4 above)
Comment (MahmoudMH) [reply src/client/components/navbar/Navbar.js]: done

File: src/client/components/admindashborad/students/index.js (hunk 3 above)
Comment (MahmoudMH) [reply src/client/components/admindashborad/students/index.js]: Sure :+1:

File: src/client/components/admindashborad/students/index.js (hunk 2 above)
Comment (MahmoudMH) [reply src/client/components/admindashborad/students/index.js]: done

File: public/assets/github.svg (hunk 1 above)
Comment (MahmoudMH) [reply public/assets/github.svg]: it's svg for github icon, which was Walaa used it

**Pull request merged**